R/utils.R

```diff
@@ -115,17 +121,31 @@ getMultipeptide <- function(precursors, features, applyFun=lapply, masters = NUL
   multipeptide
 }
 
-dummyTbl <- function(analytes){
-  data.table("transition_group_id" = analytes, "feature_id" = bit64::NA_integer64_,
-             "RT" = NA_real_, "intensity" = NA_real_, "leftWidth" = NA_real_,
-             "rightWidth" = NA_real_, "peak_group_rank" = NA_integer_, "m_score" = NA_real_)
+dummyTbl <- function(analytes, runType="DIA_Proteomics"){
+  if ( runType=="DIA_IPF" ){
+    data.table("transition_group_id" = analytes, "feature_id" = bit64::NA_integer64_,
+               "RT" = NA_real_, "intensity" = NA_real_, "leftWidth" = NA_real_,
+               "rightWidth" = NA_real_, "peak_group_rank" = NA_integer_, "ms2_m_score" = NA_real_, "m_score" = NA_real_)
```

Comment:
Should we call `ms2_m_score` something like `IPF m-score`?

Reply:
`ms2_m_score` is not the `IPF m-score`, it's the `MS2 QVALUE` which is the default `m_score` that get returned by pyprophet. When doing **IPF**, then the `m_score` gets updated based on the **IPF** framework returning updated `QVALUE`s. I've named it `ms2_m_score` to retain the old `MS2 QVALUEs`. I would like to keep this name to keep it consistent with TRIC and Pyprophet?

Reply:
Ok